corehq/apps/userreports/models.py

```diff
@@ -883,6 +884,29 @@ def bulk_creation(cls, doc_ids, doc_type, domain, config_ids):
             for doc_id in doc_ids
         ])
 
+    @classmethod
+    def bulk_update_records(cls, configs_by_docs, domain, doc_type_by_id):
+        # configs_by_docs should be a dict of doc_id -> list of config_ids
+        if not configs_by_docs:
+            return
+        doc_ids = configs_by_docs.keys()
+
+        old_indicators = AsyncIndicator.objects.filter(doc_id__in=doc_ids).all()
+        to_update = []
+
+        for indicator in old_indicators:
+            if set(indicator.indicator_config_ids) != set(configs_by_docs[indicator.doc_id]):
```

Comment:
Instead of `!=` I think that we want to see if the new config_ids are a subset of the old config_ids. If it is a subset, then we don't need to update. 

If they don't match we want the superset of both the old config ids and new config ids

Reply:
Ha, I see. Will update